api.md

```diff
@@ -0,0 +1,4 @@
+# Web API

```

Comment:
Let's name this `web_api.md` to not confuse with the python API. What information beyond the current sentence should be in here? At least a reference to the swagger UI, I suppose?

Reply:
@tdoan2010 will update this. We can either merge this now or wait for him

Reply:
Please don't wait for me. I'll open another PR when I'm finished.